app/controllers/admin/films_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Admin::FilmsController < ApplicationController
+class Admin::FilmsController < Admin::BaseController
```

Comment:
nếu để kế thừa `AppicationController` thì có được ko?

Reply:
Được nhưng mình muốn check admin cho toàn bộ file trong folder admin